.travis.yml

```diff
@@ -6,10 +6,9 @@ php:
     - 7.4
 
 env:
-    - LARAVEL_VERSION=5.6.*
```

Comment:
Why did you removed 5.6 and 5.7 versions from testing?

Reply:
They are well out of support, little point in testing them now. Meant to remove the from the `composer.json` as well.